docs/integrations/builtin/credentials/shopify.md

```diff
@@ -13,53 +13,110 @@ You can use these credentials to authenticate the following nodes with Shopify.
 - [Shopify](/integrations/builtin/app-nodes/n8n-nodes-base.shopify/)
 - [Shopify Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.shopifytrigger/)
 
-## Prerequisites
-
-- Create a [Shopify](https://shopify.com/){:target=_blank .external-link} account.
-- [Create and install a custom app](https://help.shopify.com/en/manual/apps/app-types/custom-apps){:target=_blank .external-link}.
-
 ## Supported authentication methods
 
-- API token
-- OAuth2
-- API key
+- Access token (recommended): For private apps/single store use. Can be created by regular admins.
+- OAuth2: For public apps. Must be created by partner accounts.
+- API key: Deprecated
 
 ## Related resources
 
 Refer to [Shopify's authentication documentation](https://shopify.dev/docs/apps/auth){:target=_blank .external-link} for more information about the service.
 
-## Using API token
+## Using access token
 
-To configure this credential, you'll need:
+To configure this credential, you'll need a [Shopify](https://shopify.com/){:target=_blank .external-link} admin account and:
 
-- Your **Shop Subdomain**: Your subdomain is within the URL: `https://<subdomain>.myshopify.com`. For example, if the full URL is `https://n8n.myshopify.com`, the Shop Subdomain is `n8n`.
-- An **Access Token**: Generated when you create a custom app as the **Admin API Access Token**. View it in the app's **API Credentials** section.
-- An **APP Secret Key**: Generated when you create a custom app as the **API Secret Key**. View it in the app's **API Credentials** section.
+- Your **Shop Subdomain**
```

Comment:
This would be a change from the current default (in which we only use periods after these lists if there's additional text). So I'm going to ignore this suggestion as it would introduce inconsistency for the current format, but this one might be a good one for you to contemplate for the style guide moving forward. :)

Reply:
That works for me. Thanks!